Xamarin.PropertyEditing.Mac/Controls/Custom/CommonBrushLayer.cs

```diff
@@ -68,6 +70,7 @@ public static CALayer CreateBrushLayer (CommonBrush brush)
 		public override void LayoutSublayers ()
 		{
 			base.LayoutSublayers ();
+			Frame = new CGRect (Frame.X, Bounds.Y + VerticalMargin + 0.5, Bounds.Width, Bounds.Height - VerticalMargin * 2);
```

Comment:
Is it worth making the `0.5` and `2` constants?

Reply:
Also just a preference rather than anything, I would tend to wrap the `VerticalMargin * 2` in brackets just so it's very clear that is takes precedent. I know that as programmers we should all know BIDMAS, but at a glance IMO, it just makes things clearer.

Reply:
2, certainly not (it's just reflecting that a control has two vertical ends: up and down; it's not a measure of anything, and couldn't have another meaningful value). 0.5 is a one-time fudge that is also not going to be adjusted, reasonably, but a constant could make it more explicit what it's for.